frontend/src/lib/k8s/event.ts

```diff
@@ -21,7 +21,8 @@ export interface KubeEvent {
   [otherProps: string]: any;
 }
 
-class Event extends makeKubeObject<KubeEvent>('Event') {
+class Event extends KubeObject<KubeEvent> {
+  static objectName = 'Event';
```

Comment:
The point of this objectName was to keep a consistent class name because the class names get mangled and minimized.
So I guess we should call this class name. Would there be a nicer way to define these than having a static var declared (it's fine, just wondering if there's anything alternative that is nicer).

Reply:
I've kept the objectName to avoid breaking changes